Makefile

```
@@ -113,12 +113,24 @@ clean:
 
 .PHONY: e2e
 e2e:
-	$(GO) run github.com/onsi/ginkgo/ginkgo --v --randomizeAllSpecs --randomizeSuites --race $(if $(TEST),-focus '$(TEST)') $(TAGS) ./test/e2e -- $(if $(SKIPTLS),-skip-tls true) 
+	$(GO) run github.com/onsi/ginkgo/ginkgo --v --randomizeAllSpecs --randomizeSuites --race $(if $(TEST),-focus '$(TEST)') $(TAGS) ./test/e2e -- $(if $(SKIPTLS),-skip-tls true)
 
 
 .PHONY: release
 export OPM_IMAGE_REPO ?= quay.io/operator-framework/opm
 export IMAGE_TAG ?= $(OPM_VERSION)
-release: RELEASE_ARGS?=release --rm-dist --snapshot
+export IMAGE_TAG_MAJ_MIN ?= $(shell echo $(OPM_VERSION) | grep -Eo 'v[0-9]+\.[0-9]')
+export IMAGE_TAG_MAJ ?= $(shell echo $(OPM_VERSION) | grep -Eo 'v[0-9]+')
+# LATEST_TAG is the latest semver tag in HEAD. Used to deduce whether
+# OPM_VERSION is the new latest tag, or a prior minor/patch tag, below.
+LATEST_TAG := $(shell git tag -l --sort=-v:refname | head -n1)
```

It looks like this sort function sorts prereleases _after_ non-prereleases.

This seems to work better:
```suggestion
LATEST_TAG := $(shell git tag -l | tr - \~ | sort -V | tr \~ - | tail -n1)
```

---

Nice catch.